client/blocks/disconnect-jetpack/style.scss

```diff
@@ -42,22 +40,34 @@
 	}
 }
 
-.disconnect-jetpack-dialog__button-wrap {
+.disconnect-jetpack__button-wrap {
 	margin-top: 32px;
-	margin-bottom:16px;
+	margin-bottom: 16px;
 
 	.button {
 		margin: 0 4px;
 	}
 }
 
-.disconnect-jetpack-dialog__more-info-link {
+.disconnect-jetpack__more-info-link {
 	font-size: 13px;
 }
 
+// Dialog
+
+.disconnect-jetpack-dialog .dialog__content {
+	padding: 0;
+}
+
+.disconnect-jetpack-dialog {
+	.disconnect-jetpack {
```

Comment:
I understand the reasoning behind it, but it's totally weird to see a `.disconnect-jetpack-dialog .disconnect-jetpack` selector 😲

Reply:
Yeah, I was struggling with this for a while, trying to move most of the re-usable bits to the new `DisconnectJetpack` component. Unfortunately, some of the generic `Dialog` component's selectors are being prioritized quite highly, so I needed a rather specific selector in order to override them.

Since I spent quite a bit of time last night trying to tweak this (and there's a chance we might have to iterate on this later anyway -- pending design feedback on #18796), I'm inclined to leave as-is for now...

Reply:
Yeah, let's go with it. We can always iterate in the future and rename the class.